projects/dart/features/environment.py

```diff
@@ -8,10 +8,17 @@
 from webdriver_manager.phantomjs import PhantomJsDriverManager
 
 
-BEHAVE_DEBUG_ON_ERROR = True
+BEHAVE_DEBUG_ON_ERROR = os.getenv('BEHAVE_PDB', '')
```

Comment:
This is a little confusing, as it relies on simply the presence of the environment variable (and the Pythonic blank-string-implies-False). It makes sense, but is there some place to document this I wonder?

Reply:
Yeah, this seems like it would be a candidate for an item to include in a DART style guide that we could conceivably create. I went with an empty string because even if the environment variable is set to `True`, it would resolve in Python as `"True"`.

I also wanted to avoid situations like this:
```
>>> bool('False')
True
```